changelog/15608.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+**Snowflake Database Plugin**: Adds support for managing Snowflake users with RSA key pair credentials.
```

Comment:
I wonder if this will create a duplicate entry? I added it (probably a little too early 😄) to Vault main already in a different PR https://github.com/hashicorp/vault/blob/main/changelog/15376.txt.

Reply:
oh, yeah it probably will. I can remove this one